tests/foreman/installer/test_installer.py

```diff
@@ -1306,10 +1306,28 @@ def extract_help(filter='params'):
 def common_sat_install_assertions(satellite):
     sat_version = 'stream' if satellite.is_stream else satellite.version
     assert settings.server.version.release == sat_version
+
+    # no errors/failures in journald
+    result = satellite.execute(
+        r'journalctl -q --no-pager -b -p err -u "pulpcore" -u "foreman" -u "foreman-proxy" -u "httpd" -u "postgresql" -u "pulpcore-api" -u "pulpcore-worker"'
+    )
+    assert len(result.stdout) == 0
+    # no errors/failures in /var/log/foreman/production.log
+    # ignore warnings
+    result = satellite.execute(r'grep -i "error" /var/log/foreman/production.log | grep -v "\[W\|"')
```

Comment:
the second grep should be `grep -vE "\[W\|"` so that the regexp is understood, otherwise it will always return the empty list. That being said, what about the Info lines that can contain mentions of error as well

Reply:
now I grep lines with severity "E" (error) only. and I added the extended regexp option. please check